config/software/automake.rb

```diff
@@ -15,19 +15,30 @@
 #
 
 name "automake"
-default_version "1.11.2"
+default_version "1.15"
 
 dependency "autoconf"
 
-source url: "https://ftp.gnu.org/gnu/automake/automake-#{version}.tar.gz",
-       md5: "79ad64a9f6e83ea98d6964cef8d8a0bc"
+version "1.15" do
+  source md5: "716946a105ca228ab545fc37a70df3a3"
+end
+
+version "1.11.2" do
+  source md5: "79ad64a9f6e83ea98d6964cef8d8a0bc"
+end
+
+source url: "https://ftp.gnu.org/gnu/automake/automake-#{version}.tar.gz"
 
 relative_path "automake-#{version}"
 
 build do
   env = with_standard_compiler_flags(with_embedded_path)
 
-  command "./bootstrap", env: env
+  if version.to_f >= 1.15
```

Comment:
We should use `version.satisfies?('>= 2.1')` for this. Example: https://github.com/chef/omnibus-software/blob/master/config/software/ruby.rb#L156